bootstrap-salt.sh

```diff
@@ -1442,9 +1442,13 @@ __check_dpkg_architecture() {
             if [ "$_CUSTOM_REPO_URL" != "null" ]; then
                 warn_msg="Support for arm64 is experimental, make sure the custom repository used has the expected structure and contents."
             else
-                # Saltstack official repository does not yet have arm64 metadata,
-                # use arm64 repositories on arm64, since all pkgs are arch-independent
-                __REPO_ARCH="arm64"
+                # Saltstack official repository has arm64 metadata beginning with Debian 11,
+                # use amd64 repositories on arm64 for anything older, since all pkgs are arch-independent
+                if [ "$DISTRO_NAME_L" = "debian" ] || [ "$DISTRO_MAJOR_VERSION" -lt 11 ]; then
```

Comment:
Wondering how this will handle RHEL 9 on aarch64, see https://www.centos.org/download/, supported aarch64 even back in 7, not that I ever want to build classic packages for it.

Conditional needs a little work

Reply:
@dmurphy18 This is only in the Debian section.

Reply:
This is Debian related only.

Reply:
Unless I misread the code.